src/Apps/MyCollectionInsights/Routes/MyCollectionInsightsMedianSalePriceAtAuction/MyCollectionInsightsMedianSalePriceAtAuction.tsx

```diff
@@ -0,0 +1,14 @@
+import { Box, Text } from "@artsy/palette"
+
+export const MyCollectionInsightsMedianSalePriceAtAuction = () => {
+  return (
+    <Box>
+      <Text mt={2} variant={"lg-display"}>
```

Comment:
**Issue:** According to the designs, the header should be `lg` in Desktop view and `lg-display` in mWeb
```suggestion
      <Text mt={2} variant={["lg-display", "lg"]}>
```